lib/web_ui/lib/src/engine/platform_dispatcher.dart

```diff
@@ -500,10 +500,22 @@ class EnginePlatformDispatcher extends ui.PlatformDispatcher {
             // TODO(a-wallen): As multi-window support expands, the pop call
             // will need to include the view ID. Right now only one view is
             // supported.
-            implicitView!.browserHistory.exit().then((_) {
+            //
+            // TODO(mdebbar): What should we do in multi-view mode?
+            //                https://github.com/flutter/flutter/issues/139174
```

Comment:
Should we CC a-wallen in the issue? I think both your comment and the one above are talking about the multi-window support.

Reply:
cc @a-wallen :)